### PR TITLE
Add Google Plus publisher meta

### PR DESCRIPTION
**Pull request by jonalport**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Application < Rails::Application
     config.i18n.load_path    += Dir[Rails.root.join('config', 'locales', '**/*', '*.yml').to_s]
 
     config.crazy_egg_url         = '//dnn506yrbagrg.cloudfront.net/pages/scripts/0018/4438.js'
+    config.google_plus_mas_url   = '//plus.google.com/+MoneyadviceserviceOrgUkYourMoneyAdvice/'
```
Comment (benbarnett) [inline config/application.rb]: catchy

Comment (jonalport) [inline config/application.rb]: I try my best

**Comment by asmega** [comment]
marevelous :+1:

**Comment by jonalport** [comment]
Thanks for the enthusiasm, Phil!

**Pull request merged**